ParabolInc/action
# Redirect to original url after login

This PR addresses https://github.com/ParabolInc/action/issues/436

Test steps:
- npm run dev
- login
- start a meeting
- copy the url for the meeting
- log out during the meeting
- paste meeting url in browser
- log in
=> should be redirected to the meeting url you pasted

@mattkrick @jordanh @ackernaut
